### DISCUSSION TITLE
About native of scrollbar

### DISCUSSION THREAD

**Discussion by kooriookami** [element-plus/element-plus#1034] [Q&A]
What is the native style?
```html
<el-scrollbar native>
  ...
</el-scrollbar>
```
No scrollbar like this?
![image](https://user-images.githubusercontent.com/38392315/102564942-6e914700-4117-11eb-939c-1adc6b353f6c.png)
Or native scrollbar like this?
![image](https://user-images.githubusercontent.com/38392315/102564977-82d54400-4117-11eb-9712-e34da6a917c3.png)
I'd like to fix some problem about scrollbar.

**Comment by jw-foss** [comment]
![image](https://user-images.githubusercontent.com/15975785/102565818-2d019b80-4119-11eb-9fe3-e6f1d6a74e7e.png)
I believe we implemented this scrollbar just like MacOS's native scroll bar.

**Reply by jw-foss** [reply]
![image](https://user-images.githubusercontent.com/15975785/102565870-473b7980-4119-11eb-9733-db812c437d75.png)
The activated state.
The previous screenshot is the style of half-activated (which is about to disappear)

**Reply by kooriookami** [reply]
Is that the `native` using the browser's own scrollbar style? 
My understand:
`native: true`: using browser's scrollbar style.
`native: false`: using element-plus scrollbar style.
Do I understand it right?

**Reply by jw-foss** [reply]
Yep. that's right.

**Reply by jw-foss** [reply]
When the `native` flag is false, we use styles like the screenshots.

**Reply by kooriookami** [reply]
I made a fix, you could review it. #1042
